framework/assets/yii.js

```diff
@@ -272,12 +272,13 @@ yii = (function ($) {
         },
 
         getQueryParams: function (url) {
+            url = url || document.location.href;
```

Comment:
[No!](http://www.nooooooooooooooo.com/) :)
Share your thoughts, please. What is your use case?